Several fixes

jbreu opened the pull request.

Galixte committed:
FRENCH translation for Reputation System extension

FRENCH translation for exceptions.php file
FRENCH translation for info_acp_reputation.php
FRENCH translation for permissions_reputation.php
FRENCH translation for reputation_acp.php
FRENCH translation for reputation_common.php
FRENCH translation for reputation_rating.php
FRENCH translation for reputation_system.php
FRENCH translation for reputation_toplist.php
FRENCH translation for reputation_warning.php

Galixte committed:
FRENCH translation for info_acp_reputation.php file

FRENCH translation for info_acp_reputation.php file

Galixte committed:
FRENCH translation for permissions_reputation.php file

FRENCH translation for permissions_reputation.php file

Galixte committed:
FRENCH translation for reputation_acp.php file

FRENCH translation for reputation_acp.php file

Galixte committed:
FRENCH translation for reputation_common.php file

FRENCH translation for reputation_common.php file

Galixte committed:
FRENCH translation for reputation_rating.php file

FRENCH translation for reputation_rating.php file

Galixte committed:
FRENCH translation for reputation_system.php file

FRENCH translation for reputation_system.php file

Galixte committed:
FRENCH translation for reputation_toplist.php file

FRENCH translation for reputation_toplist.php file

Galixte committed:
FRENCH translation for reputation_warning.php file

FRENCH translation for reputation_warning.php file

Galixte committed:
Update for FRENCH translation for info_acp_reputation.php file

Update for FRENCH translation for info_acp_reputation.php file

Galixte committed:
Update for ENGLISH translation for info_acp_reputation.php file

Update for ENGLISH translation for info_acp_reputation.php file

Galixte committed:
Update for FRENCH translation - UTF8 FRENCH quotes

Update for FRENCH translation for reputation_acp.php file

Galixte committed:
Update for ENGLISH translation for reputation_system.php file

Update for ENGLISH translation for reputation_system.php file

Source: https://www.phpbb.com/community/viewtopic.php?p=13706431#p13706431

Galixte committed:
Update for FRENCH translation for reputation_system.php file

Update for FRENCH translation for reputation_system.php file

Source: https://www.phpbb.com/community/viewtopic.php?f=456&t=2210021&start=45#p13706431

Galixte committed:
Minor update for ENGLISH for reputation_acp.php file

Minor update for ENGLISH for reputation_acp.php file

Galixte committed:
Update for reputation_common.php file

Update for reputation_common.php file : Change a language key which already exists (duplicate) in reputation_system.php

Galixte committed:
Update for reputation_common.php file

Update for reputation_common.php file which already exists (duplicate) in reputation_system.php

Galixte committed:
Correct update for reputation_common.php file

Correct update for reputation_common.php file in relation with: https://www.phpbb.com/community/viewtopic.php?f=456&t=2210021&start=120#p13851641

Galixte committed:
Correct update for reputation_system.php file

Correct update for reputation_system.php file in relation with: https://www.phpbb.com/community/viewtopic.php?f=456&t=2210021&start=120#p13851641

Galixte committed:
Correct update for reputation_common.php file

Correct update for reputation_common.php file in relation with: https://www.phpbb.com/community/viewtopic.php?f=456&t=2210021&start=120#p13851641

Galixte committed:
Add missing L_COLON keys

Galixte committed:
fix correct class

Galixte committed:
Fix colors class and padding button on viewtopic

Galixte committed:
fix colors class and padding button on viewtopic

Galixte committed:
Fix padding button on viewtopic

Galixte committed:
Update of FRENCH translation for Reputation System extension

Galixte committed:
Update of FRENCH translation for Reputation System extension

Galixte committed:
Update of FRENCH translation for Reputation System extension

Galixte committed:
Update of FRENCH translation for Reputation System extension

Galixte committed:
Update of FRENCH translation for Reputation System extension

Galixte committed:
Update of FRENCH translation for Reputation System extension

Galixte committed:
Update of FRENCH translation for Reputation System extension

Galixte committed:
Update of FRENCH translation for Reputation System extension

Galixte committed:
Update of FRENCH translation for Reputation System extension

Galixte committed:
Update padding on viewtopic buttons

Commit:
Working code for phpBB 3.2

WorkingWarrior committed:
Fixes in notifications system

Commit:
3.2 fixes: fixed parent constructor call in notifications and update yml according to symfony update

jbreu committed:
Merge branch 'rhea_fixes' into rhea_fixes

jbreu committed:
Merge pull request #1 from WorkingWarrior/rhea_fixes

Fixes in notifications system

jbreu committed:
Minor fixes due to merging issues

jbreu committed:
Add German translation (not in acp)

jbreu committed:
Remove icons (replaced by fontawesome icons); fix post reputation display

jbreu committed:
Fix buttons when post reputation is given

jbreu committed:
Push version to 1.1.0-dev

jbreu committed:
Merge branch 'patch-2' of https://github.com/Galixte/phpBB-Reputation-System into Galixte-patch-2

jbreu committed:
Update Readme.md

jbreu committed:
Update README.md

Galixte committed:
Bad names of language keys

Galixte committed:
Bad names of language keys

Galixte committed:
Bad names of language keys

Galixte committed:
missing colon

Galixte committed:
missing colon

Galixte committed:
missing colon

jbreu committed:
Merge pull request #3 from Galixte/patch-3

Bad names of language keys and missing colons

Galixte commented on language/pl/reputation_system.php:
```diff
@@ -0,0 +1,119 @@
+	'RS_POWER_DETAILS'				=> 'Sposób obliczania mocy reputacji',
+	'RS_POWER_DETAIL_AGE'			=> 'Według daty rejestracji',
+	'RS_POWER_DETAIL_POSTS'			=> 'Według liczby postów',
+	'RS_POWER_DETAIL_REPUTAION'		=> 'Według reputacji',
```
@jbreu: Should be ```RS_POWER_DETAIL_REPUTATION``` i have fixed this in english & french languages.

Galixte commented on language/de_x_sie/reputation_system.php:
```diff
@@ -0,0 +1,116 @@
+	'RS_POWER_DETAILS'				=> 'Wie die Reputation berechnet wird',
+	'RS_POWER_DETAIL_AGE'			=> 'Durch Registrierungsdatum',
+	'RS_POWER_DETAIL_POSTS'			=> 'Durch die Anzahl der Beiträge',
+	'RS_POWER_DETAIL_REPUTAION'		=> 'Durch die Reputation',
```
@jbreu: Same, it should be ```RS_POWER_DETAIL_REPUTATION``` i have fixed this in english & french languages.

Galixte commented on language/pl/info_acp_reputation.php:
```diff
@@ -0,0 +1,58 @@
+	'LOG_POST_REPUTATION_CLEARED'	=> '<strong>Usunięto reputację posta</strong><br />Autor posta: %1$s<br />Temat posta: %2$s',
+	'LOG_USER_REPUTATION_CLEARED'	=> '<strong>Usunięto reputację użytkownika</strong><br />Użytkownik: %1$s',
+	'LOG_REPUTATION_SYNC'			=> '<strong>System reputacji ponownie zsynchronizowany</strong>',
+	'LOG_REPUTATION_TRUNCATE'		=> '<strong>Reputacja wyczyszczona</strong>',
```
@jbreu, add on a new line this missing language key: ```'REPUTATION_SETTINGS_CHANGED'	=> '<strong>Altered Reputation System settings</strong>',``` with your translation, please.

Galixte replied:
Or maybe, this message concerns @WorkingWarrior ?

Galixte commented on language/de_x_sie/info_acp_reputation.php:
```diff
@@ -0,0 +1,58 @@
+	'LOG_POST_REPUTATION_CLEARED'	=> '<strong>Beitragsbewertung gelöscht</strong><br />Autor des Beitrags: %1$s<br />Beitragsthema: %2$s',
+	'LOG_USER_REPUTATION_CLEARED'	=> '<strong>Benutzerbewertung gelöscht</strong><br />Benutzer: %1$s',
+	'LOG_REPUTATION_SYNC'			=> '<strong>Reputationssystem resynchronisiert</strong>',
+	'LOG_REPUTATION_TRUNCATE'		=> '<strong>Alle Bewertungen gelöscht</strong>',
```
@jbreu, same add on a new line this missing language key: ```'REPUTATION_SETTINGS_CHANGED'	=> '<strong>Altered Reputation System settings</strong>',``` with your translation, please.

Commit:
Language fixes thanks to @Galixte ; Polish translation tbd.

jbreu commented:
Thanks for the reports!

KotM committed:
Russian translation

KotM committed:
Mistyping

KotM committed:
More applicable translation

KotM committed:
Simplified

KotM committed:
BugFix: Incorrect reputation view link from viewtopic

SpeakerCZ commented:
In the case of a positive evaluation, the reputation points of the total available quantity are deducted; in the case of a negative evaluation, reputational points are added to the total available quantity.

An example of a positive rating
Reputation points 10 out of a total of 10
I rate a post or user positively with 5 reputation points
The balance is 5 points out of a total of 10

An example of a negative rating
Reputation points 10 out of a total of 10
I evaluate the post or user negatively with 5 reputation points
The balance is 15 points out of a total of 10

But should not 5 points out of 10 ?

Galixte commented:
Hi @SpeakerCZ,

i use this extension. I didn’t notice this behavior on my board, you can test it, here: http://www.ezcom-fr.com (select the EN flag if your are not french).

Maybe your settings are not corrects (coherent). If you want, i can share mine in english with screenshots.

Galixte commented:
@eeeman1, i suspect an other extension which is related with avatars that’s the cause of your trouble.

SpeakerCZ commented:
Hello Galixte,
thank you for being a member of your forum. I'm already registered. We have sent you a private message.

Galixte commented:
Ok, i found the same issue like you.

I think the problem is around this part of code: https://github.com/Pico/phpBB-Reputation-System/blob/master/controller/rating_controller.php#L781

The problme is the subtraction https://github.com/Pico/phpBB-Reputation-System/blob/master/controller/rating_controller.php#L758
```
$voting_power_left = $max_voting_power - $used_power;
```
Which means:
voting power (A) = maximum allowed voting power (B) - voting power used (C)

BUT it should calculate like this:  A =  B -  C even if =C is negative, for example:
- if B = 20 and C = 5, so:
A = B - C ==> OK A = 15
- if B = 20 and C = -5, so:
A = B - (-C) ==> NOT OK A = 25 it should = 15

It takes into account the negative value and adds this with the subtraction thus -- = +.

Any solution @jbreu or @WorkingWarrior  ?

jbreu commented:
I never used that feature. But reading your comments a quick solution seems to be to use the absolute numbers instead of the actual pos/neg values: A = B - abs(C)

bobalo2 commented:
Ugh , I'm confused. How do I get this to work with phpbb3.2 ? Can I download the files that were fixed or do I have to change them one by one?

jbreu commented:
you may want to download one of the current forks: https://github.com/Pico/phpBB-Reputation-System/network

Galixte commented:
For phpBB 3.2.x the last version is here: https://github.com/jbreu/phpBB-Reputation-System/archive/master.zip.

jbreu committed:
Merge pull request #5 from KotM/Russian-translation

Russian translation

jbreu committed:
Merge pull request #6 from KotM/Bugfixes

Incorrect reputation view link

Fredhoob commented:
I've been using the 1.1.0-dev pico version on my forum for a while. Could I just replace the files with yours and keep all my previous settings, votes of my members, etc.? Thanks.

jbreu commented:
It should work, but a backup of all data is always recommended

Commit:
Fixes to make it run on phpBB3.3
Credits to mrgoldy: https://www.phpbb.com/community/viewtopic.php?p=15393276#p15393276

rbonick committed:
Included comments in notifications

rbonick committed:
Remove accidental file copy

rbonick committed:
Fix reputation icons not showing

rbonick committed:
Fix reputation comments not being censored

jbreu committed:
Merge pull request #9 from rbonick/styles

Fix reputation icons not showing

jbreu committed:
Merge pull request #10 from rbonick/censor-comments

Fix reputation comments not being censored

jbreu committed:
Merge pull request #8 from rbonick/master

Adds reputation comments to the notifications themselves

jbreu committed:
fix compatibility with phpBB 3.3.10

fix compatibility with phpBB 3.3.10 , i.e. the newer versions of twig contained in it.
